runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -446,6 +481,13 @@ public synchronized CorfuRuntime connect() {
                 throw new RuntimeException(e);
             }
         }
+
+        if (loadSmrMapsAtConnect) {
+            FastSmrMapsLoader fastLoader = new FastSmrMapsLoader(this)
+                    .setBatchReadSize(bulkReadSizeForFastLoader)
+                    .setTimeoutInMinutesForLoading(timeoutInMinutesForFastLoading);
+            fastLoader.loadMaps();
+        }
```

Comment:
actually, thinking about this more: connect doesn't return the runtime until after loadMaps() completes, so actually there is no risk of activity under loadMaps() is done, no? Am I missing something?